docs/source/noise/index.rst

```diff
@@ -38,45 +36,55 @@ Currently, errors added in different columns are independent of each other.
 Available Noise Types
 ---------------------
 
-Pseudopeople applies the different types of row-based and column-based noise in
-the following order to mimic the data generation process by which a real dataset
-might be corrupted. The table lists all the available noise types, but not every
+The table lists all the available noise types, but not every
 type of noise will necessarily be applied to every dataset or every column. The
 configuration determines which noise types are actually used. The "Noise Type"
 column shows the name of the noise type in the configuration system.
 
-.. list-table:: Types of noise in order of application
-  :widths: 1 2 5 1
+.. list-table:: Types of row-based noise (``row_noise``)
+  :widths: 1 2 5 
   :header-rows: 1
 
   * - Noise Type
-    - Description
-    - Example
-    - Order
-  * - ``row_noise``
-    - **Row-based noise**
-    - ---
-    - ---
-  * - ``omit_row``
-    - Omit a row
+    - Config key 
+    - Example cause
+  * - Omit a row
+    - ``omit_row``
     - Neglecting to file a tax form on time
-    - 1
-  * - ``column_noise``
-    - **Column-based noise**
-    - ---
-    - ---
-  * - ``leave_blank``
-    - Leave a field blank
+
+.. list-table:: Types of column-based noise (``column_noise``)
+  :widths: 1 2 5 
+  :header-rows: 1
+
+  * - Noise Type
+    - Config key
+    - Example cause
+  * - Leave a field blank
+    - ``leave_blank``
     - Forgetting to write your name on the designated line
-    - 2
-  * - ``choose_wrong_option``
-    - Choose the wrong option from a fixed set of options
+  * - Choose the wrong option
+    - ``choose_wrong_option``
     - Marking the "Male" box when you meant "Female"
-    - 3
-
-.. todo::
+  * - Optical character recognition (OCR) error
+    - ``make_ocr_errors``
+    - Misreading an 'S' instead of a '5'
+  * - Make typos
+    - ``make_typos``
+    - Accidentally typing an 'l' instead of a 'k' because they are 
+      right next to each other on a QWERTY keyboard
+  * - Misreport age
+    - ``misreport_age``
+    - Reporting that you are 28 years old when you are actually 27
+  * - Write the wrong digits
+    - ``write_wrong_digits``
+    - Writing "2022" instead of "2002" in your date of birth
+  * - Write the wrong zipcode digits 
+    - ``write_wrong_zipcode_digits``
+    - Writing ZIP code 98118 when you actually live in 98112
```

Comment:
Yeah it might be a bit confusing as-is but I don't see a good way to explain this without adding another column to this table. Especially when we link from this table directly to the detail sections, a reader would only be one click away from the explanation.

Reply:
> Especially when we link from this table directly to the detail sections

Oh, is that the plan? E.g., turn the names of the noise functions into hyperlinks to the appropriate sections? That sounds like a good idea. In that case, we should probably at least add a note to the description of "write wrong zipcode digits" highlighting how it's different from "write wrong digits," and maybe we should add a note to "write wrong digits" as well.